pom.xml

```diff
@@ -1888,7 +1888,8 @@
                     <overWriteSnapshots>false</overWriteSnapshots>
                     <overWriteIfNewer>true</overWriteIfNewer>
                     <excludeGroupIds>org.apache.hadoop,org.apache.hbase,org.apache.hive,com.google.protobuf,\
-                      org.datanucleus,asm,org.apache.spark,slf4j-log4j12,org.codehaus.jackson</excludeGroupIds>
+                      org.datanucleus,asm,org.apache.spark,slf4j-log4j12,org.codehaus.jackson,\
+                      org.apache.tez</excludeGroupIds>
```

Comment:
Do you know what brings in tez-api dependency? It's good to find out which one and be able to exclude it from the transitive dependency if possible.